marco-gabe/public/scripts/models/repo.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+var app = app || {};
+
+(function(module) {
+  const repos = {};
+
+  repos.all = [];
+
+  repos.requestRepos = function(callback) {
+    // TODO: How would you like to fetch your repos? Don't forget to call the callback.
+    //       Remember that the callback function we'll want to call relies on repos.all
+    //       being an array with a bunch of repo objects in it, so you'll need to
+    //       populate it with the response from Github before you call the callback.
+    $.get('/github/user/repos', function(response) {
+      repos.all = response.map(function(repo) {return repo})
```

Comment:
No need to map them, the response we will get back is an array of objects(repos) which is already what we want inside repos.all.